scripts/circle_deploy.sh

```diff
@@ -10,26 +10,18 @@ git config --global user.name "tadeploy"
 git config --global user.email "devteam@technologyadvice.com"
 
 #
-# build
-#
-gulp docs
-
-#
 # generate changelog
 #
 echo "...generating changelog"
-github_changelog_generator $CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME
+curl -X POST github-changelog-api.herokuapp.com/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME \
```

Comment:
I had seen it used somewhere.  Had to double check this one.  Found it in their [docs](https://circleci.com/docs/configuration) under configuration as a valid way to install phantom.js.

Keep in mind the build passing here is only testing our components.  The easiest way to test this deploy script, is to deploy it :)